packages/datetime/src/dateinput.md

```diff
@@ -51,11 +48,17 @@ function getMomentFormatter(format: string): IDateFormatProps {
 <DateInput {...getMomentFormatter("LL")} locale="de" />
 ```
 
+@## Props
```

Comment:
Write out the word "properties" for professionalism.

Reply:
"props" is actually a react convention, and they're never called "properties" (to distinguish from JS objects)

Reply:
see React's own docs: https://reactjs.org/docs/components-and-props.html